src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -2406,4 +2410,26 @@ public static void closeErrorDialog(RemoteRobot remoteRobot) {
             okButton.click();
         }
     }
+
+    /**
+     * Cleans up resources and stops the Liberty server if necessary.
+     *
+     * @param remoteRobot       The RemoteRobot instance.
+     * @param absoluteWLPPath   The absolute path of the Liberty installation.
+     * @param projectName       The name of the project to select.
+     * @param testName          The name of the test calling this method.
+     */
+    public static void cleanupAndStopServerIfNeeded(RemoteRobot remoteRobot, String absoluteWLPPath, String projectName, String testName) {
+
+        // Open the terminal window.
+        openTerminalWindow(remoteRobot);
```

Comment:
why do we need to open the terminal window here?

Reply:
Since the openFile method has `hideTerminalWindow(remoteRobot);`, the terminal window gets minimised while reading the server.env file. So inorder to bring it back to the frame, i added the open terminal window method there.

Reply:
Removed the method `cleanupAndStopServerIfNeeded` from UIBotTestUtils.java. Just added the method to `runStopAction` only in `testCustomStartParametersClearedOnConfigRemoval` method itself.